freq: .5
speed: .025
perlin_joints: ['HeadYaw', 'HeadPitch']
joint_names: ['HeadYaw', 'HeadPitch', 'LShoulderPitch',
'LShoulderRoll', 'LElbowYaw', 'LElbowRoll', 'LWristYaw',
'LHand', 'LHipYawPitch', 'LHipRoll', 'LHipPitch', 'LKneePitch',
 'LAnklePitch', 'LAnkleRoll', 'RHipYawPitch', 'RHipRoll', 'RHipPitch',
  'RKneePitch', 'RAnklePitch', 'RAnkleRoll', 'RShoulderPitch',
   'RShoulderRoll', 'RElbowYaw', 'RElbowRoll', 'RWristYaw', 'RHand']
